frontend/src/pages/Common/History/HistorySearch.js

```diff
@@ -70,7 +70,7 @@ const HistorySearch = ({ classes, fetchFirstHistoryEvents, users, eventTypes })
       <DatePicker
         id="filter-endat"
         className={classes.datepicker}
-        label={"end"}
+        label={strings.history.endDate}
         name="endAt"
```

Comment:
`strings.history.endAt` is missing in all language files

Reply:
since without a word in this area, a style bug occurs (the position of the date delete button is not accurate). This style bug should be resolved after setting `strings.history.endAt`